lib/versioning/api.ts

```diff
@@ -15,6 +15,7 @@ import * as npm from './npm';
 import * as nuget from './nuget';
 import * as pep440 from './pep440';
 import * as poetry from './poetry';
+import * as pvp from './pvp';
```

Comment:
I think this is a new feature, but I'm not seeing any user-facing documentation in this PR. How will our users know/learn about this new feature?

Reply:
When should we write user-facing documentation given that we are implementing this as [3 different PR](https://github.com/renovatebot/renovate/issues/8187#issuecomment-758552305)s? So far this just adds the versioning.

Or would it be better to revert this change to this file until it's feature complete?

Reply:
You don't need to revert the changes in this file. 😉 

We could maybe add a `lib/versioning/pvp/readme.md` file, like the Docker versioning has for example: https://github.com/renovatebot/renovate/tree/main/lib/versioning/docker

@viceice Will the Renovate docs generating code automatically "know about" the `pvp` versioning once the PR lands, or is more work needed to get the `pvp` versioning to show up in this section of our docs? https://docs.renovatebot.com/modules/versioning/#supported-versioning

Reply:
I've added a readme. Let me know if there should be any other information I should include. Thanks!